### ISSUE TITLE
Replace the google form contact form with a dedicated email on a /contact page

### ISSUE THREAD

**Issue by kingjacob** [quantstamp/quantstamp_web_support#10]
## Expected Behavior
The contact form to be under the quantstamp domain.

## Actual Behavior
A linnk to a google form (https://docs.google.com/forms/d/e/1FAIpQLScGoZq3acsrAOP_7Ex_mgOi174_QoMYR04HqvNM79hNIgSWIQ/viewform)

## Steps to Reproduce the Problem
This can be improved by 

1. Self hosting a contact form under quantstamp.com/contact
2. Embedding the google form under quanstamp.com/contact
3. Doing what stripe does below and just post a team@quantstamp.com email mailto:link (provided you block robots from scraping emails)

3 is the Recommendation because a dedicated contact page and email would also be a good way to guide people to a knowledge page or support center as stripe does, which would decrease the amount of manual support requests you have. And organize incoming request better via something like HelpScout.

## Examples
 
<img width="1161" alt="screen shot 2018-05-01 at 10 29 27 am" src="https://user-images.githubusercontent.com/806312/39484421-a2942358-4d2a-11e8-9c37-2852a99d292d.png">

**Comment by MeiMcCullar** [comment]
Good point. We've been talking about this one for awhile. It's on our list to do! @Matt-Holland @JaxDesign
